alert/us.go

```diff
@@ -0,0 +1,703 @@
+package alert
+
+// USCERT alerts
```

Comment:
__[Golint]__ <sub>posted by Sider</sub>

comment on exported var AlertDictUS should be of the form "AlertDictUS ..."